imports/plugins/included/ui-search/client/templates/searchModal/searchModal.js

```diff
@@ -1,212 +1,28 @@
-import _ from "lodash";
+import $ from "jquery";
```

Comment:
I think we want this to be `import { $ } from "meteor/jquery"; so we aren't pulling in another copy of jQuery?